semantic-source/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - Adds an `NFData` instance for `Source`.
 
+- Decodes to `Text` leniently instead of throwing exceptions.
```

Comment:
We should try to get into the habit of adding notes to changelogs in PRs for this package.